gradle.properties

```diff
@@ -1,3 +1,10 @@
+android.enableJetifier=true
```

Comment:
Why is this set to true?

Reply:
`enableJetifier ` collocate with `useAndroidX ` to enable AndroiX

Reply:
Right but we don't need jetifier because we don't have any legacy support library dependencies so it should be set to false.